packages/web-components/src/link/link.ts

```diff
@@ -0,0 +1,40 @@
+import { attr } from '@microsoft/fast-element';
+import { BaseAnchor } from '../anchor-button/anchor-button.js';
+import { StartEnd } from '../patterns/index.js';
+import { applyMixins } from '../utils/apply-mixins.js';
+import { type LinkAppearance } from './link.options.js';
+
+/**
+ * An Anchor Custom HTML Element.
+ * Based largely on the {@link https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a | <a> element }.
+ *
+ * @slot start - Content which can be provided before the link content
+ * @slot end - Content which can be provided after the link content
+ * @slot - The default slot for link content
+ *
+ * @public
+ */
+export class Link extends BaseAnchor {
+  /**
+   * The appearance the link should have.
+   *
+   * @public
+   * @remarks
+   * HTML Attribute: `appearance`
+   */
+  @attr
+  public appearance?: LinkAppearance | undefined;
+
+  /**
+   * The link is inline with text
+   * In chromium browsers, if the link is contained within a semantic
+   * text element (`h1`, `h2`, `h3`, `h4`, `h5`, `h6`, `p`) or `fluent-text`,
+   * `:host-context()` ensures inline links are styled appropriately.
+   *
+   * @public
+   * @remarks
+   * HTML Attribute: `inline`
+   */
+  @attr({ mode: 'boolean' })
+  public inline?: boolean = false;
```

Comment:
fwiw, I assumed `inline` was for controlling display. it took me awhile to figure out it was about inheriting styles when used as phrasing content. can't think of a better word though.

Reply:
If it's per the design spec, then I think we roll with it until it needs to change. The lack of cross-browser `:host-context()` is frustrating.

Reply:
Yeah, I'd love to see broader adoption of that, it's a huge benefit for CE's in some scenarios.